services/web-app/src/lib/auth.ts

```diff
@@ -1,17 +1,34 @@
-import { NextAuthOptions } from "next-auth"
-import GithubProvider from "next-auth/providers/github"
+import { DynamoDB, DynamoDBClientConfig } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocument } from "@aws-sdk/lib-dynamodb"
+import { DynamoDBAdapter } from "@next-auth/dynamodb-adapter"
+import { NextAuthOptions } from "next-auth";
+import GithubProvider from "next-auth/providers/github";
+import { clientId, clientSecret, awsAccessKeyId, awsSecretKey, awsRegion } from "./config";
 
-const clientId = process.env.GITHUB_ID;
-const clientSecret = process.env.GITHUB_SECRET;
+export const dynamoConfig: DynamoDBClientConfig = {
```

Comment:
So talking with Matt, it seems like you should be able to put the dynamodb table in the same stack as the lambda, and then you won't need this because it'll have the correct permission